sdk/src/loom2d/Display/Graphics.ls

```diff
@@ -44,29 +34,66 @@ package loom2d.display
     };
     
     [Native(managed)] 
+    /**
+     * Describe the format of some text for the Graphics class.
+     */
     public native class TextFormat
     {
+        /**
+         * Describe a font format by specifying font name, indicating size, color, and if it's bolded.
+         */
         function TextFormat(font:String = null, size:Number = NaN, color:Number = NaN, bold:Object = null) {
             if (font != null) this.font = font;
             if (!isNaN(size)) this.size = size;
             if (!isNaN(color)) this.color = color;
         }
+        
+        /**
+         * What font should we use with this text format?
+         */
         public native function set font(value:String);
         public native function get font():String;
+        
+        /**
+         * RGB color of the text.
+         */
         public native function set color(value:int);
         public native function get color():int;
+        
+        /**
+         * Size of text in points.
+         */
         public native function set size(value:float);
         public native function get size():float;
+        
+        /**
+         * Align as specified in TextAlign.
+         */
         public native function set align(value:int);
         public native function get align():float;
+        
+        /**
+         * Add additional spacing between letters.
+         */
         public native function set letterSpacing(value:float);
         public native function get letterSpacing():float;
+        
+        /**
+         * Add additional space between lines (in pixels).
+         */
         public native function set lineHeight(value:float);
         public native function get lineHeight():float;
+        
+        /**
+         * Load a font from a given path and register it under the specified name.
```

Comment:
Only certain font files are supported at the moment, at least one of TTF. See nanovg / fontstash / stb_truetype for details.